ark/segmentation/marker_quantification.py

```diff
@@ -5,16 +5,133 @@
 
 import xarray as xr
 
-from skimage.measure import regionprops_table
+from skimage.measure import regionprops, regionprops_table
 
 from ark.utils import io_utils, load_utils, misc_utils, segmentation_utils
-from ark.segmentation.signal_extraction import extraction_function
+from ark.segmentation.signal_extraction import EXTRACTION_FUNCTION
+from ark.segmentation.regionprops_extraction import REGIONPROPS_FUNCTION
 
 import ark.settings as settings
 
 
+def compute_extra_prop_info(prop_info, regionprops_extras, **kwargs):
+    """Derives new features specified by regionprops_extras from a regionprops features
+
+    Args:
+        prop_info (skimage.measure.regionprops):
```

Comment:
lets use props instead of prop_info to be more concise and consistent with regionprops naming